AbletonMCP_Remote_Script/handlers/devices.py

```
@@ -0,0 +1,1366 @@
+                    sample_data[prop] = val
+                elif isinstance(val, (int, float)):
+                    sample_data[prop] = val
+                elif hasattr(val, '__int__'):
+                    sample_data[prop] = int(val)
+                else:
+                    sample_data[prop] = str(val)
+            except Exception:
+                sample_data[prop] = None
+        try:
+            sample_data["file_path"] = str(sample.file_path)
+        except Exception:
```

Ruff (0.15.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)